src/mame/matsushita/kn5000.cpp

```diff
@@ -591,6 +618,16 @@ void kn5000_state::machine_start()
 	save_item(NAME(m_mstat));
 	save_item(NAME(m_sstat));
 
+	if(m_extension)
+	{
```

Comment:
The existing code in this file (as well as the majority of MAME) uses a space between flow control keywords and the opening parenthesis of the controlling expression, like `if (whatever)`.